timesketch/frontend/src/components/Sketch/TimelineListItem.vue

```diff
@@ -135,6 +135,32 @@ limitations under the License.
       Added {{ timeline.updated_at | moment("YYYY-MM-DD HH:mm") }}
       <span class="tag is-small" :title="meta.stats[timeline.searchindex.index_name]['count'] + ' events in index'">{{ meta.stats[timeline.searchindex.index_name]['count'] | compactNumber }}</span>
     </span>
+
+    <div v-if="timelineStatus === 'ready'" class="is-size-6 small-top-margin">
+      <a @click="toggleExpansion(timeline)">
```

Comment:
You can simplify this a bit by using buefy. We are supporting Buefy components that implements some useful logic and UI interactions. See: https://buefy.org/documentation/collapse

Reply:
I'll leave it as a todo for another merge request :)

Reply:
👍